docs/docs/integrations/chat/seekrflow.ipynb

```diff
@@ -0,0 +1,385 @@
+     "output_type": "stream",
+     "text": [
+      "Hello there! I'm Seekr, nice to meet you! What brings you here today? Do you have a question, or are you looking for some help with something? I'm all ears (or rather, all text)!\n"
+     ]
+    }
+   ],
+   "source": [
+    "response = llm.invoke([HumanMessage(content=\"Hello, Seekr!\")])\n",
+    "print(response.content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "853b0349",
+   "metadata": {},
+   "source": [
+    "## Chaining"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "35fca3ec",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "content='The translation of \"Good morning\" in French is:\\n\\n\"Bonne journée\"' additional_kwargs={} response_metadata={}\n"
+     ]
+    }
+   ],
+   "source": [
+    "prompt = ChatPromptTemplate.from_template(\"Translate to French: {text}\")\n",
+    "\n",
+    "chain: RunnableSequence = prompt | llm\n",
+    "result = chain.invoke({\"text\": \"Good morning\"})\n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "a7b28b8d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "🔹 Testing Sync `stream()` (Streaming)...\n",
+      "Here is a haiku:\n",
+      "\n",
+      "Golden sunset fades\n",
+      "Ripples on the quiet lake\n",
+      "Peaceful evening sky"
+     ]
+    }
+   ],
+   "source": [
+    "def test_stream():\n",
+    "    \"\"\"Test synchronous invocation in streaming mode.\"\"\"\n",
+    "    print(\"\\n🔹 Testing Sync `stream()` (Streaming)...\")\n",
+    "\n",
+    "    for chunk in llm.stream([HumanMessage(content=\"Write me a haiku.\")]):\n",
+    "        print(chunk.content, end=\"\", flush=True)\n",
+    "\n",
+    "\n",
+    "# ✅ Ensure streaming is enabled\n",
+    "llm = ChatSeekrFlow(\n",
+    "    client=seekr_client,\n",
+    "    model_name=\"meta-llama/Meta-Llama-3-8B-Instruct\",\n",
+    "    streaming=True,  # ✅ Enable streaming\n",
+    ")\n",
+    "\n",
+    "# ✅ Run sync streaming test\n",
+    "test_stream()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c81e70ad",
+   "metadata": {},
+   "source": [
+    "## Handling Stop Tokens"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "b32f3041",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "In th\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Using a stop token to truncate output\n",
+    "messages = [HumanMessage(content=\"Tell me a story about a dragon.\")]\n",
+    "response = llm.invoke(messages, stop=[\"e\"])\n",
+    "\n",
+    "print(response.content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b3847b34",
+   "metadata": {},
+   "source": [
+    "## Error Handling & Debugging"
```

Comment:
Please remove this section -- it's not at all helpful for debugging or testing -- it's all mocks.